Formula/virgil.rb

```diff
@@ -0,0 +1,27 @@
+class Virgil < Formula
+  desc "CLI tool to manage your Virgil account and applications"
+  homepage "https://github.com/VirgilSecurity/virgil-cli"
+  url "https://github.com/VirgilSecurity/virgil-cli.git",
+     :tag      => "v5.0.1",
+     :revision => "03ddee65e9cecbdd7ef55b1285c0ca70758d6d40"
+  head "https://github.com/VirgilSecurity/virgil-cli.git"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/VirgilSecurity/virgil-cli"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+    cd dir do
+      system "dep", "ensure", "-vendor-only"
+      system "go", "build", "-o", "virgil"
+      bin.install "virgil"
+    end
+  end
+
+  test do
+    result = `virgil pure keygen`
```

Comment:
This is not a good way to call the function. Please use `shell_output()` like the other formulae.